conf/nomad/minio.hcl

```diff
@@ -137,6 +137,24 @@ MINIO_SECRET_KEY="${secret_key}"
 ${ envs }
 EOF
       }
+%{ if use_vault_kms }
+template {
```

Comment:
Think this should have the same indentation as the other templates

Reply:
Something like the updated one?